### PR TITLE
Flexbox profile

### PR DESCRIPTION
**Pull request by PacoSchwab**
CSS styling :)

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -18,11 +18,20 @@ body {
...
+  display: flex;
+  flex-direction: column;
 }
 
 .tech-stack__list {
   list-style: none;
   padding-left: 0;
+  display: flex;
+  gap: 10px;
+  flex-wrap: wrap;
```
Comment (toshydev) [inline css/styles.css:34]: You can add `justify-content: space-between;` to place the `<li>` elements.

**Review by toshydev — CHANGES_REQUESTED** [review]
One issue, otherwise good work! 👌

File: css/styles.css (hunk 1 above)
Comment (PacoSchwab) [reply css/styles.css:34]: > You can add `justify-content: space-between;` to place the `<li>` elements.

As an alternative to "gap"? Yes, that`s possible, bust as the idea is to create gaps of at least 10px, "gap" works fine as well, right?

Comment (toshydev) [reply css/styles.css:34]: True👌

**Pull request merged**